src/main/java/frc/robot/RobotContainer.java

```diff
@@ -74,19 +75,44 @@ public double getAsDouble() {
     private class Turn implements DoubleSupplier {
         @Override
         public double getAsDouble() {
-            return -m_xbox.getRightX();
+            if (Robot.isReal())
+                return m_xbox.getRightX();
+            else
+                return m_xbox.getLeftX();
```

Comment:
First, you lost a minus sign on the controller value. And 2nd, no, leave the controller the same in the simulation.